src/local-history.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from 'vscode';
+import path = require('path');
+import fs = require('fs');
+
+// export interface FileProperties {
+//     fileName: string;
+//     uri: string;
+//     lastModified: string;
+//     fileHistory?: string[];
+// }
+
+export interface historyFileProperties {
+    fileName: string;
+    timeStamp: string;
+    uri: string;
+}
+
+export class LocalHistoryProvider {
+
+    private historyFiles: any[] = [];
+    /**
+     * View all history of the active editor
+     */
+    // eslint-disable-line no-unused-vars
+    public viewAllForActiveEditor(textEditor: vscode.TextEditor): void {
```

Comment:
Question: Since we are not using `textEditor`, will we not run into some issues when looking up the local history of a specific file?

Reply:
It was actually looking up all the local history files. I have updated the code to fix this.